web_external/Viewer/index.js

```diff
@@ -140,7 +140,7 @@ class Viewer extends PureComponent {
                                         g0
                                     }
                                 })}
-                                annotationDeleted={() => this.props.dispatch({
+                                deleteAnnotation={() => this.props.dispatch({
```

Comment:
This is not consistent with other names, especially `annotationDrawn`. If we're going to make changes, I would prefer making a general change, not an incomplete change that decreases consistency.

Reply:
I am naming this based on what actually happened from the redux store (Or VAUI) point of view. The difference here is for annotationDrawn, it is drawn with GeoJS, it is something already happened, In the store, you could choose to ignore the drawn result but can't revert the fact that an annotation was drawn. 
But for annotation delete, from the store's point of view,  it was a request to delete. Naming it annotationDeleted sounds like it already happened. 
It just happens that GeoJS internally call that annotation, VAUI calls the boxes annotations as well. So, I think I would rename annotationDrawn to rectangleDrawn, at least for now, it is the only kind we support.